device/iothub-java-client/src/main/java/com/microsoft/azure/iothub/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -503,6 +518,7 @@ public void onReactorFinal(Event event)
                 openAsync();
             } catch (IOException e)
             {
+                logger.LogError(e);
                 e.printStackTrace();
             }
         }
```

Comment:
Could add appropriate logging in each of these override methods using debug level ? At the very least provide a succesful entry logging.